content/blog/fish-extract.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "`extract` in fish"
+description = ""
+tags = ["development", "fish", "shell"]
+categories = ["Development"]
+# series = []
+date = 2019-11-14T10:40:55-05:00
+draft = false
++++
+I recently switched from `zsh` to `fish` as my shell of choice.
+I liked the idea of starting from scratch, with the sane defaults that `fish` provides, as my [`zsh` configuration files were getting a bit out of control](https://github.com/svanburen/dotfiles/blob/master/zshrc).
+
+For the most part, the transition was fairly painless and straightforward.
+However, I still missed a few of the niceties that I had been used to from `zsh` over the years - one of which was the [`extract` function provided in oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/extract/extract.plugin.zsh).
```

Comment:
LINE 14 - PERHAPS RECONFIGURE SENTENCE.  EX: "HOWEVER, I STILL MISS A FEW OF THE 'ZSH' NICETIES THAT I HAVE GROWN ACCUSTOM TO OVER THE YEARS - ONE BEING"

Reply:
thanks - fixed